style.css

```diff
@@ -0,0 +1,65 @@
+
+body {
+    margin: 0px;
+    font-family: 'Roboto', sans-serif;
+}
+
+advantages {
+    max-width: 1440px;
+    max-height: 569px;
+}
+
+.advantages-header {
+    left:567px;
```

Comment:
Because of using `position: relative`, and `left`
You have a horizontal scroll on your web-site
Please don't use `position: relative` or `absolute` without reasons, 
you should be sure, that there are not any options except positioning
I recommend     `text-align: center;`  instead of `position` here
![image](https://user-images.githubusercontent.com/25488411/156524953-922023f3-5526-4722-84f1-ab27c22f51ea.png)